templates/cassiopeia/scss/blocks/_global.scss

```diff
@@ -1,25 +1,14 @@
 // Global
 :root {
   scroll-behavior: smooth;
+  font-size: $font-size-root;
```

Comment:
I dont remember exactly the issue number at the moment but there was a very specific reason that this was set on html and not root.

Reply:
https://github.com/joomla/joomla-cms/search?q=%22font-size-root%22&type=issues
https://github.com/joomla/joomla-cms/search?q=%22font-size-base%22&type=issues

I don't see an issue where states that it will not work on `:root`
It was implemented directly on `html` with the creation of Aurora => https://github.com/joomla/cassiopeia/commit/4da1d9ed25ee9ad041df5b3fa14bdcb1a2a3dd14

Reply:
it works on root of course but I am sure I recalled a discussion that it doesnt work on root in a certain circumstance. I guess we will find oout during testing